app/controllers/Application.java

```diff
@@ -430,6 +430,7 @@ public static Result flowHistory() {
     if (results.size() == 0) {
       return notFound("Unable to find record on flow url: " + flowDefId);
     }
+    IdUrlPair flowDefPair = new IdUrlPair(flowDefId, results.get(0).flowDefUrl);
```

Comment:
Anywhere in the code where you use it is fine(like the location of the comment). Just want to keep this info to be findable in the code.